image/image_linux.go

```diff
@@ -144,6 +171,36 @@ func installCloudConfig(rootDir, gadgetDir string) error {
 	return osutil.CopyFile(cloudConfig, dst, osutil.CopyFlagOverwrite)
 }
 
+func customizeImage(rootDir, defaultsDir string, custo *Customizations) error {
+	// customize with cloud-init user-data
+	if custo.CloudInitUserData != "" {
+		varCloudDir := filepath.Join(rootDir, "/var/lib/cloud/seed/nocloud-net")
```

Comment:
It'd be great to include a reference to those doc pages in the comment:
https://cloudinit.readthedocs.io/en/latest/topics/dir_layout.html
https://cloudinit.readthedocs.io/en/latest/topics/datasources/nocloud.html

Reply:
I will add this in a follow up